estate/models/estate_property.py

```diff
@@ -0,0 +1,120 @@
+from odoo import api, fields, models
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools.float_utils import float_is_zero, float_compare
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = 'All properties'
```

Comment:
```suggestion
    _description = 'Estate Properties'
```
Nitpick :innocent: 
Also description should always be in title case

Reply:
Good point. All updated for all models :smiling_face_with_tear: